### PR TITLE
Temporarily remove lettuce CI

### PR DESCRIPTION
**Pull request by gchenfc**
Lettuce CI fails because camera models with distortion are not supported.

Temporarily taking this out since it's not on anyone's A->B.  (See https://github.com/borglab/gtsfm/pull/540#issuecomment-1229300655)

Potential fixes:
* Support other camera models
* rerun colmap with a simple camera model for baseline/gt (IIRC, this dataset is from plant 018 "experiment 3" (Spring 2022) and that camera should have not had much distortion, i.e. it wasn't wide-angle/fisheye)

### CODE REVIEW

**Review by kfu02 — APPROVED** [review]
LGTM, assuming the remaining CI checks pass

**Comment by johnwlambert** [comment]
Let's not merge this yet -- I think I have a 5 line fix for this.

**Comment by johnwlambert** [comment]
Please take a look at https://github.com/borglab/gtsfm/pull/558

**Comment by gchenfc** [comment]
#558 fixes lettuce CI so no need to remove it anymore

**Pull request closed without merging**
